src/main/java/org/jitsi/videobridge/EndpointConnectionStatus.java

```diff
@@ -404,4 +360,64 @@ public void handleEvent(Event event)
             .filter(e -> e.getConference() == conference)
             .forEach(e -> sendEndpointConnectionStatus(e, false, endpoint));
     }
+
+    /**
+     * How long it can take an endpoint to send first data before it will
+     * be marked as inactive.
+     */
+    public static class FirstTransferTimeoutProperty extends AbstractConfigProperty<Long>
+    {
```

Comment:
If these two properties are both scoped under `videobridge.ep-connection-status` in the config file, would it make sense to put them under the same master property object (like you did with `HealthConfig`)?

Reply:
`EndpointConnectionStatus` is at the top level, so didn't wan to add a catch-all `config` package there, but do you mean just within the `EndpointConnecitonStatus` class itself?  like doing:

```
class EndpointConnectionStatus
{
  public static class Config
  {
    public static class FirstTransferTimeoutProperty
    {
    }
  }
}
```
?

Do you think it would add much? I could do it...don't feel strongly either way.

Reply:
You don't like having an `EndpoingConnectionStatusConfig` class or something like that because it's at the top level?

Reply:
`EndpointConnectionStatus` is in `org.jitsi.videobridge` and should probably be moved, as it's just a weird dumping ground for stuff, so i thought adding a `config` package there with config for all those classes would only add to the disarray, but i can do it.

Reply:
Up to you, you have a better big-picture view of how everything's fitting together than I do.

Reply:
ok, so i went to go add it, but i think it'll be misleading to have a `config` package at the top with all the other packages, and it didn't really feel 'connected' with the files at the top level, so it seemed confusing to me.  i think the long term fix here is to better organize the files and put the stuff at the top level in sub-packages

Reply:
thought about this some more last night, and do think it'd be slightly better if i nested a `Config` static class here with the properties inside that...that way it's at least more consistent.  so will take a look at that today.